esp/src/src-react/components/FileDetails.tsx

```diff
@@ -148,7 +149,8 @@ export const FileDetails: React.FunctionComponent<FileDetailsProps> = ({
             </PivotItem>
             <PivotItem headerText={nlsHPCC.XML} itemKey="XML" style={pivotItemStyle(size, 0)}>
             </PivotItem>
-            <PivotItem headerText={nlsHPCC.Superfiles} itemKey="Superfiles" style={pivotItemStyle(size, 0)}>
+            <PivotItem headerText={nlsHPCC.Superfiles} itemKey="superfiles" itemCount={file?.Superfiles?.DFULogicalFile.length || 0} style={pivotItemStyle(size, 0)}>
```

Comment:
Noticed this when making changes from the previous review. The || 0 should be an ok default, if something is undefined?

Reply:
Yes that is fine.

Reply:
(As long as you want (0) to be displayed?